SimPEG/EM/Base.py

```diff
@@ -126,6 +126,19 @@ def MfMui(self):
             self._MfMui = self.mesh.getFaceInnerProduct(self.mui)
         return self._MfMui
 
+    def MfMuiDeriv(self, u):
+        """
+        Derivative of :code:`MfMui` with respect to the model.
+        """
+        if (
```

Comment:
minor comment on style:

I would return Zero early. It is the easy case to exclude and allows you to whittle down assumptions and get to the hard things last.